public/push-analytics.json

```diff
@@ -28,23 +40,47 @@
                             "YEAR_OVER_YEAR_COLUMN",
                             "SCATTER",
                             "BUBBLE",
-                            "SINGLE_VALUE",
-                            "PIVOT_TABLE",
-                            "OUTLIER_TABLE"
+                            "SINGLE_VALUE"
                         ],
-                        "selector": ".highcharts-container"
+                        "selector": ".highcharts-container > svg"
```

Comment:
I'm not sure I understand this correctly, but I don't think this selector is present for SINGLE_VALUE?
Single value does not use Highcharts for rendering the SVG.

Reply:
Good point, solved now!